### ISSUE TITLE
Can't install budou using pip

### ISSUE THREAD

**Issue by stevenle** [google/budou#32]
Quick isolated case:

```
virtualenv venv
source venv/bin/activate
pip install budou
```

I get error traceback:

```
Collecting budou                                   
  Using cached budou-0.6.0.tar.gz                  
    Complete output from command python setup.py egg_info:                                             
    Traceback (most recent call last):             
      File "<string>", line 20, in <module>        
      File "/private/var/folders/01/w01_1_fx077_0zpzqxm2092m0000gn/T/pip-build-9UqicV/budou/setup.py", line 31, in <module>                                                                                   
        install_requires=read_file('requirements.txt').splitlines(),                                   
      File "/private/var/folders/01/w01_1_fx077_0zpzqxm2092m0000gn/T/pip-build-9UqicV/budou/setup.py", line 19, in read_file                                                                                  
        with open(os.path.join(os.path.dirname(__file__), name), 'r') as f:                            
    IOError: [Errno 2] No such file or directory: '/private/var/folders/01/w01_1_fx077_0zpzqxm2092m0000gn/T/pip-build-9UqicV/budou/requirements.txt'                                                          
                                                   
    ----------------------------------------       
    Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/01/w01_1_fx077_0zpzqxm2092m0000gn/T/pip-build-9UqicV/budou    
```

**Comment by tushuhei** [comment]
Thank you for reporting. I will work on this on priority.

**Comment by tushuhei** [comment]
I uploaded the packaged to PyPi in wheel format, so the issue should be fixed now.

**Comment by ylsm** [comment]
Happening again with 0.8.1

```
Collecting budou
  Using cached budou-0.8.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-7Eu7J6/budou/setup.py", line 31, in <module>
        install_requires=read_file('requirements.txt').splitlines(),
      File "/tmp/pip-build-7Eu7J6/budou/setup.py", line 19, in read_file
        with open(os.path.join(os.path.dirname(__file__), name), 'r') as f:
    IOError: [Errno 2] No such file or directory: '/tmp/pip-build-7Eu7J6/budou/requirements.txt'
```

**Comment by tushuhei** [comment]
Thank you for flagging. This is fixed in 0.8.3, so please try it again.
